templates/footer.html

```diff
@@ -0,0 +1,21 @@
+<footer class="text-center">
+  <div id="base">
+    <div class="container">
+
+	<p id="quote"></p>
+
+      <p class="copy">Designed by <br>
+        <a href="http://www.harrymt.com/" target="_blank">harrymt</a> for <a class="hacksoc" href="http://www.hacksocnotts.co.uk/">hacksoc</a>
+      </p>
+      <ul class="social">
+        <li class="facebook"><a href="https://www.facebook.com/witnotts" target="_blank">Facebook</a></li>
+        <li class="twitter"><a href="https://twitter.com/inspireWIT" target="_blank">Twitter</a></li>
+      </ul>
+    </div>
+
+</footer>
+
+<script src="main.min.js"></script>
+
+</body>
```

Comment:
I see your point. The index file is now auto generated from Grunt but I could move it into a new `end.html` file so it would just sit at the end of the concatenated file? The footer file could then easily be removed or changed.